import type { AxiosError } from 'axios';
export default function isRetryAllowed(error: AxiosError): boolean;
